### ISSUE TITLE
lazyload in horizontal carousel

### ISSUE THREAD

**Issue by czy0729** [sergiodxa/react-lazy-image#4]
```
<SwipeableViews 
    className={`${prefixCls}__swiper`} 
    index={state.index}
    animateTransitions={state.animateTransitions}
    onChangeIndex={index => this.swiperChange(index)}
>
    {
        state.data.map((item, idx) => (
            <Image 
                className={`${prefixCls}__img`} 
                key={idx}
                source={item.url}
            />
        ))
    }
</SwipeableViews>
```
I have a Carousel but  it's moving horizontally , so it just the first img has lazy and other do not work.

[http://ww3.sinaimg.cn/mw690/715ed77dgw1f9g6x6iyj9g20b00cqkf1.gif](url)

if there some trigger api ? 

By the way my english is very bad, i hope you can help me, thank you!

**Comment by czy0729** [comment]
I has saw the origin code, and I find the constructor has not bind the fetch() 

I change the code so I can get the fetch method by ref. 

and your code is very beautiful and I has learn many.

**Comment by sergiodxa** [comment]
I understood your use case, if you already make it works (and it still works without doing anything extra in normal cases) you can send a pull request, I'm going to review it and merge it without any problem :)

**Comment by sergiodxa** [comment]
Closed in favor of #14
